src/natcap/invest/urban_flood_risk_mitigation.py

```diff
@@ -677,7 +677,7 @@ def _runoff_retention_op(q_pi_array, p_value, q_pi_nodata, result_nodata):
     result = numpy.empty_like(q_pi_array)
     result[:] = result_nodata
     valid_mask = numpy.ones(q_pi_array.shape, dtype=numpy.bool)
-    if q_pi_nodata:
+    if isinstance(q_pi_nodata, (int, float)):

```

Comment:
Ahhh i see, I feel like this is more obvious what's happening here if you could change this for a test for `None`like this:

```
if q_pi_nodata is not None:
```

This way it shows that `q_pi_nodata` is allowed to be None. Testing if it's a numeric seems like a circular way to do that and isn't obvious why it wouldn't be numeric.